libs/docs/platform/table/examples/advanced-scrolling/advanced-scrolling-example.component.html

```diff
@@ -8,14 +8,19 @@
     emptyTableMessage="No data found"
     bodyHeight="300px"
     [virtualScroll]="true"
+    [enhancedRendering]="enhancedRendering"
     [pageScrolling]="true"
-    [pageSize]="50"
-    [renderAhead]="50"
+    [pageSize]="20"
+    [renderAhead]="20"
     [pageScrollingThreshold]="80"
     hasChildrenKey="hasChildren"
     [expandOnInit]="true"
 >
-    <fdp-table-toolbar title="Order Line Items" [hideItemCount]="false" [hideSearchInput]="true"> </fdp-table-toolbar>
+    <fdp-table-toolbar title="Order Line Items" [hideItemCount]="false" [hideSearchInput]="true">
+        <fdp-table-toolbar-actions *ngIf="enhancedRendering">
```

Comment:
`enhancedRendering` is default `false` in the `advanced-scrolling-example.component.ts` . Do we need add a button to toggle `enhancedRendering` in this example?

Reply:
I think there is no need to toggle as it may cause issue (page freeze) if there are many columns in the example. The example should show-case how it works with new change as there is another table on the same page for comparison.